### PR TITLE
bump some icesat2 env versions

### PR DESCRIPTION
**Pull request by jhamman**
cc @scottyhq

### CODE REVIEW

File: deployments/icesat2/image/binder/environment.yml [hunk 1]
```diff
@@ -16,18 +16,19 @@ dependencies:
...
   - h5py=2.9
   - h5netcdf=0.7
-  - dask=1.1
-  - distributed=1.25
-  - dask-kubernetes=0.8
+  - dask=2.3
+  - distributed=2.3
+  - dask-kubernetes=0.9.2
   - matplotlib=3.1
   - nbserverproxy=0.8
```
Comment (andersy005) [inline deployments/icesat2/image/binder/environment.yml:33]: `nbserverproxy` got rename to `jupyter-server-proxy`

File: deployments/icesat2/image/binder/environment.yml [hunk 2]
```diff
@@ -16,18 +16,19 @@ dependencies:
   - nomkl
   - s3fs=0.2
```
Comment (andersy005) [inline deployments/icesat2/image/binder/environment.yml:17]: There's a more recent release of s3fs too

File: deployments/icesat2/image/binder/environment.yml [hunk 3]
```diff
@@ -16,18 +16,19 @@ dependencies:
...
   - gcsfs=0.2
-  - intake=0.4
+  - intake=0.5
```
Comment (andersy005) [inline deployments/icesat2/image/binder/environment.yml:19]: Do we need to specify `intake` as a dependency? I think that having `intake-xarray` takes care of this

**Comment by scottyhq** [comment]
fine by me, but before merging this let's add a profile list so that hackweek attendees can checkout the image with an exact copy of packages used during the hackweek. See (https://github.com/pangeo-data/pangeo-cloud-federation/blob/70580aa714b4a4b9d5375467f98f5c7e92ebe5ca/deployments/nasa/config/common.yaml#L61). We can point to the image that I copied over to DockerHub - `uwhackweeks/icesat2:2019`.

**Comment by jhamman** [comment]
@scottyhq - I have to run. Can you both make those changes in separate PRs?

**Pull request merged**